src/service/runnerService.ts

```diff
@@ -1,10 +1,61 @@
 import { resolve as res } from 'path';
 import * as vscode from 'vscode';
+import { CodeUtil } from '../common/codeUtil';
 import { FileManager, FileModel } from '../common/fileManager';
 import { ConfigKey, Global } from '../common/global';
 import { Process } from '../common/processWrapper';
 
 export class RunnerService {
+    /** Align assignments and comments in selection. */
+    public static async alignSelection(): Promise<void> {
+        const editor = vscode.window.activeTextEditor;
+        if (!editor) {
+            vscode.window.showErrorMessage('No active editor found!');
+            return;
+        }
+
+        const document = editor.document;
+        const selection = editor.selection;
+        // User can select part of first and/or last line of text. For example:
+        //      this text not selected [this text selected on first line
+        //      this text selected on second line] this text is not selected
+        // 'selection.end' is position of last character in selection,
+        // not last character of second line! Same problem with start position of 'selection.start'.
+        // Get position of last character in second line.
+        const endLinePosition = document.lineAt(selection.end.line).range.end;
+        const range = document.validateRange(
+            new vscode.Range(
+                selection.start.line,
+                0,
+                selection.end.line,
+                endLinePosition.character,
```

Comment:
What is `validateRange` for, if not to allow us to reasonably use `Infinity` 😆 this is mostly a joke, but this is also technically simpler code

```suggestion
                Infinity,
```